nc/views.py

```diff
@@ -104,7 +104,9 @@ def query(self, results, group_by, filter_=None):
             to_date = datetime.datetime.strptime(_to_date, "%Y-%m-%d")
             if from_date and to_date:
                 # Add another month to include the last month chosen
-                to_date = to_date + relativedelta.relativedelta(months=1)
+                to_date = (to_date + relativedelta.relativedelta(months=1)) - datetime.timedelta(
```

Comment:
Updating this to get data up to but not including the next month. To prevent the table data from showing incorrect stats, including the first day of the extra month.